.prettierignore

```diff
@@ -0,0 +1,8 @@
+.github/
+.yarn/
+.yarnrc.yml
+.gitpod.yml
+.prettierrc
+next-env.d.ts
+README.md
+renovate.json
```

Comment:
@MatrixFrog **question:** Is there a reason to exclude the dot files and readme and such?  On the API project it looks like it's mostly just the auto-generated files.

https://github.com/distributeaid/aggregated-public-information/blob/saga/.prettierignore

Reply:
I guess that's true. the next-env file says it shouldn't be edited but anything that's JSON is probably fine to format.